src/Components/Facility/PatientNoteCard.tsx

```diff
@@ -0,0 +1,101 @@
+import { relativeDate, formatDateTime, classNames } from "../../Utils/utils";
+import { USER_TYPES_MAP } from "../../Common/constants";
+
+export interface NoteType {
+  note: string;
+  facility: Facility;
+  created_by_object: CreatedByObject;
+  user_type: string;
+  created_date: string;
+}
+
+export interface Facility {
+  id: string;
+  name: string;
+  local_body: number;
+  district: number;
+  state: number;
+  ward_object: WardObject;
+  local_body_object: LocalBodyObject;
+  district_object: DistrictObject;
+  state_object: StateObject;
+  facility_type: FacilityType;
+  read_cover_image_url: any;
+  features: any[];
+  patient_count: number;
+  bed_count: number;
+}
+
+export interface WardObject {
+  id: number;
+  name: string;
+  number: number;
+  local_body: number;
+}
+
+export interface LocalBodyObject {
+  id: number;
+  name: string;
+  body_type: number;
+  localbody_code: string;
+  district: number;
+}
+
+export interface DistrictObject {
+  id: number;
+  name: string;
+  state: number;
+}
+
+export interface StateObject {
+  id: number;
+  name: string;
+}
+
+export interface FacilityType {
+  id: number;
+  name: string;
+}
+
+export interface CreatedByObject {
+  id: number;
+  first_name: string;
+  username: string;
+  email: string;
+  last_name: string;
+  user_type: string;
+  last_login: string;
+}
```

Comment:
We already have types defined for facility, ward, and all elsewhere in the codebase. You can reuse that instead of duplicating the types here.
```suggestion
```